sat/cli/bootsys/mgmt_power.py

```diff
@@ -260,6 +261,54 @@ def finish_shutdown(hosts, username, password, ncn_shutdown_timeout, ipmi_timeou
             sys.exit(0)
 
 
+def set_next_boot_device_to_disk(ssh_client, ncns):
+    """
+    Set the next boot device to a disk for a list of nodes via SSH.
+
+    This function connects to each node via SSH, retrieves the current boot order, identifies the first available disk,
+    and sets the next boot device to the specified disk.
+
+    Args:
+        ssh_client (paramiko.SSHClient): An active SSH client used to connect to the nodes.
+        ncns (list): A list of node names (NCNs) to configure.
+    """
+    command = "efibootmgr"
+
+    for ncn in ncns:
+        try:
+            ssh_client.connect(ncn)
+        except (SSHException, socket.error) as err:
+            LOGGER.warning(f'Unable to connect to node {ncn}: {err}')
+            continue
+
+        try:
+            _, stdout, stderr = ssh_client.exec_command(command)
+            exit_code = stdout.channel.recv_exit_status()
+            if exit_code == 0:
+                boot_order = stdout.read().decode()
+                match = re.search(r'^Boot([0-9A-Fa-f]{4})\*?\s.*UEFI OS', boot_order, re.MULTILINE)
+                if not match:
+                    LOGGER.warning(f'No disk boot entries found for {ncn}')
+                    continue
+
+                next_boot = match.group(1)
+
+                # Set the next boot device
+                next_boot_disk = f"efibootmgr -n {next_boot}"
+                _, _, stderr = ssh_client.exec_command(next_boot_disk)
+                error = stderr.read().decode()
+                if error:
+                    LOGGER.warning(f'Failed to set next boot device for {ncn}: {error}')
+                    continue
+
+                LOGGER.info(f'Successfully set next boot device to disk (Boot{next_boot}) for {ncn}')
+
+        except Exception as e:
```

Comment:
It can be a little dangerous to catch any exception which is a subclass of `Exception`. Are you expecting any other exceptions here? If not, I think it would be okay to get rid of this exception catching to avoid hiding information about any bugs in our logic.